### ISSUE TITLE
New Sitemaps got ignored when added to Eclipse-IDE

### ISSUE THREAD

**Issue by Jochen1980** [openhab/openhab-core#585]
I've installed the Eclipse-IDE and started the Jetty server with 'Standard' installation. Afterwards I was able to see BasicUI and PaperUI at localhost. Then I tried to add a sitemap under 'demo conf sitemaps'. I added and edited the corresponding file and now I was expecting, that if I call the BasicUI of localhost that I will be able to pick my new sitemap. I rechecked that after a new clean-build-cycle - nothing has changed. The new sitemap just got ignored, and in the logs there was no message like i.e. in the logviewer, which is available on port 9001 after adding this addon.

Best regards,
Jochen

**Comment by openhab-5iver** [comment]
The Eclipse Installer options have changed... [check the doc](https://www.openhab.org/docs/developer/development/ide.html). There are now two options (Addons and Core), and it sounds like you selected both, but they are not compatible with each other.

Demo will only show up after choosing Core Development (option 2). If you're seeing UIs, which are not currently working for Core Development, you must have Addons too. Install again, following the option 1 path, and see if that helps. Your sitemap would go under Infrastructure> distro-resources>...>sitemaps.

**Comment by Jochen1980** [comment]
I read the docs - thanks for the advice - indeed I checked all of the options and that was wrong of course. 

Anyway the IDE setup went wrong ... once again. 

CoreDev IDE setup: it broke because one task org.openhab.core.jar:jar starts over and over again - seems like a never ending loop or somewhat similar. 

AddonDev IDE setup: it broke on the execution tasks with a few exceptions, that some of the addons were not able to be downloaded. Currently I try it again.

**Comment by openhab-5iver** [comment]
> CoreDev IDE setup: it broke because one task org.openhab.core.jar:jar starts over and over again - seems like a never ending loop or somewhat similar.

This was mentioned [here](https://github.com/openhab/openhab-core/pull/467#issuecomment-458444910). I have not changed my JDK, but just disabled autobuild.

> AddonDev IDE setup: it broke on the execution tasks with a few exceptions, that some of the addons were not able to be downloaded. Currently I try it again.

I just recently saw this too, but everything came down properly after running the setup tasks again.

**Comment by Jochen1980** [comment]
Thanks openhab-5iver,

I can second that - after the second run (AddonDevelopment-Eclipse-IDE-setup) of the setup-tasks it works.

Regarding the Core-Dev-Eclipse-IDE setup - how can I disable the auto-build?

Cheers,
Jochen

**Comment by openhab-5iver** [comment]
> Unfortunately there is only the BasicUI visible after picking the Standard Edition of openHAB.

I just installed another fresh Addons IDE. There are some ugly errors in the log, but I have both Paper and Basic when selecting Standard.  There is only one error, which is for Paper UI about an xsd and xml thing about missing white space. At one point in getting going again I had to clear my m2, so maybe that would help, but I'd try restarting Eclipse and running the setup tasks again. If that doesn't work, try asking in the forum because it looks to be working properly.

> Regarding the Core-Dev-Eclipse-IDE setup - how can I disable the auto-build?

![image](https://user-images.githubusercontent.com/21967853/52921511-ec9b2180-32e5-11e9-9f5f-a3276abef2d8.png)
